pkg/handlers/generic/lifecycle/utils/utils.go

```diff
@@ -219,3 +222,14 @@ func CreateConfigMapForCRS(configMapName, configMapNamespace string,
 	cm.Data[defaultCRSConfigMapKey] = string(utilyaml.JoinYaml(l...))
 	return cm, nil
 }
+
+func SetTLSConfigForHelmChartProxyIfNeeded(hcp *caaphv1.HelmChartProxy) {
+	if strings.Contains(hcp.Spec.RepoURL, "mindthegap") {
+		hcp.Spec.TLSConfig = &caaphv1.TLSConfig{
+			CASecretRef: &corev1.SecretReference{
+				Name:      "mindthegap-tls",
```

Comment:
Maybe we can get this value from some env via the [downward API](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/)

Reply:
the namespace - sure. although this is a fixed namespace.

Reply:
added a change